apollo-portal/src/main/resources/static/i18n/en.json

```diff
@@ -289,6 +289,7 @@
   "Component.Publish.OpPublish": "Release",
   "Component.Rollback.To": "roll back to",
   "Component.Rollback.Tips": "This operation will roll back to the last released version, and the current version is abandoned, but there is no impact to the currently editing configurations. You may view the currently effective version in the release history page",
+  "Component.RollbackTo.Tips": "This operation will roll back to this released version, and the current version is abandoned, but there is no impact to the currently editing configurations",
```

Comment:
Should we keep 'You may view the currently effective version in the release history page'?